collections/_article/dev-snapshot-godot-4-5-dev-3.md

```diff
@@ -0,0 +1,111 @@
+
+---
+
+*The cover illustration is from* [**Cornerpond**](https://store.steampowered.com/app/3454590/Cornerpond/?curator_clanid=41324400), *a fishing game that takes place entirely in the corner of your desktop. It is developed by [foolsroom](https://foolsroom.itch.io/). You can get the game [on Steam](https://store.steampowered.com/app/3454590/Cornerpond/?curator_clanid=41324400) and follow the developer on [Twitter](https://twitter.com/foolsroom).*
+
+## Highlights
+
+In case you missed them, see the [4.5 dev 1](/article/dev-snapshot-godot-4-5-dev-1/) and [4.5 dev 2](/article/dev-snapshot-godot-4-5-dev-2/) release notes for an overview of some key features which were already in that snapshot, and are therefore still available for testing in dev 3.
+
+### Screen reader support
+
+Accessibility should be every developer's top priority, full-stop. Someone being excluded from an experience for factors outside of their control is an area that video games and applications have the potential to circumvent entirely. It does, however, take a solid framework to allow such accommodations to take place. To streamline this process for everyone—players and developers alike—our resident tech guru [bruvzg](https://github.com/bruvzg) took to the absolutely Herculean task of integrating [AccessKit](https://github.com/AccessKit/accesskit) to Godot as a whole.
+
+[GH-76829](https://github.com/godotengine/godot/pull/76829) was a project two years in the making, changing over **32,000** lines of code, and incurred hundreds of comments with feedback/testing. Users are encouraged to look at the pull request for more information, as there's no feasible way we could properly summarize these changes. Unsurprisingly, this was *by far* the change with the most ramifications of the entire snapshot, so much so that it's already seen [multiple](https://github.com/godotengine/godot/pull/105197) [fixes](https://github.com/godotengine/godot/pull/105216) caused by its regressions, but it's well worth it. After all, accessibility is our top priority!
+
+### Script backtracing
+
+In any other snapshot, this would've been the main highlight, as adding backtracing to GDScript was among the most highly requested features from our users for years. Two brave souls, [Mikael Hermansson](https://github.com/mihe) (godot-jolt) and [Juan Linietsky](https://github.com/reduz) (up-and-coming developer), helmed this task and made this process possible with [GH-91006](https://github.com/godotengine/godot/pull/91006). This will make it much easier for users to find the cause of warnings/errors that previously required manually hunting down bugs. Stack traces are now available in projects exported in release mode as well if the **Debug > Settings > GDScript > Always Track Call Stacks** project setting is enabled. This can make it easier for users to report issues in a way that developers can track down.
+
+![Backtrace script](/storage/blog/dev-snapshot-godot-4-5-dev-3/backtrace-script.webp)
```

Comment:
Screenshot is a bit outdated (from the older PR iteration), it should be `print(Engine.capture_script_backtraces())` not `print(script_backtrace())`.